lib/onc_certification_g10_test_kit/smart_ehr_patient_launch_group_stu2.rb

```diff
@@ -0,0 +1,94 @@
+module ONCCertificationG10TestKit
+  class SMARTEHRPatientLaunchGroupSTU2 < SMARTAppLaunch::EHRLaunchGroupSTU2
+    title 'EHR Launch with Patient Scopes'
+    description %(
+      # Background
+
+      If an application launched from an EHR requests and is granted a clinical
+      scope restricted to a single patient, the EHR SHALL establish a patient in
+      context.
+
+      # Test Methodology
+
+      Inferno will attempt an EHR Launch with a clinical scope restricted to a
+      single patient and verify that a patient id is received.
+
+      For more information on the #{title}
+
+      * [Apps that launch from the
+        EHR](http://hl7.org/fhir/smart-app-launch/STU2/scopes-and-launch-context.html#apps-that-launch-from-the-ehr)
+    )
+    id :g10_ehr_patient_launch_stu2
+    run_as_group
+
+    config(
+      inputs: {
+        client_id: {
+          name: :ehr_patient_client_id
+        },
+        client_secret: {
+          name: :ehr_patient_client_secret
+        },
+        requested_scopes: {
+          name: :ehr_patient_requested_scopes,
+          default: 'launch openid fhirUser offline_access patient/Patient.rs',
```

Comment:
So ... `Patient.rs` format in STU2 is the only difference? :sweat_smile:

Reply:
The only difference in scopes. The STU1 and 2 versions inherit from different base groups.